tests/integration/models/reddit/test_message.py

```diff
@@ -42,6 +42,21 @@ def test_block(self, _):
                 assert False, 'no message found'
             message.block()
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_delete(self, _):
+        self.reddit.read_only = False
+        subject = 'Test message from TestMessage.test_delete in PRAW tests'
+        with self.recorder.use_cassette('TestMessage.test_delete'):
+            self.reddit.user.me().message(subject, 'delete!')
```

Comment:
Can we simply assume there is a message to delete rather than create a message to delete? The old PRAW PRAW<4 tests use to attempt to set everything up prior to testing, but that was done before betamax was used. See `mark_unread` for an example.